charts/ciso-assistant-next/templates/ingress/tls-secret.yaml

```
@@ -2,9 +2,10 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "ciso-assistant.fullname" . }}-tls
+   name: {{ include "ciso-assistant.fullname" . }}-tls
   labels:
     {{- include "ciso-assistant.labels" (dict "context" . "component" "frontend") | nindent 4 }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 5 but found 4

(indentation)